chemprop/featurizers/atom.py

```diff
@@ -18,10 +20,15 @@ def __call__(self, a: Atom) -> np.ndarray:
         """featurize the atom ``a``"""
 
 
-@dataclass(repr=False, eq=False, slots=True)
 class MultiHotAtomFeaturizer(AtomFeaturizer):
-    """An :class:`AtomFeaturizer` featurizes atoms based on the following attributes:
+    """An :class:`MultiHotAtomFeaturizer` uses a multi-hot encoding to featurize atoms.
+
+    This featurizer provides three configurations:
+    * default
+    * v1
+    * organic
```

Comment:
This isn't correct. The featurizer does not "provide" three configurations- it's infinitely configurable. It does have classmethods which supply preset configurations, but these already have their own docstrings. You can link to them in the class-level docstring if you think that would be helpful.